### PR TITLE
Refactor aws gh actions

### PR DESCRIPTION
**Pull request by pascalwilbrink**

### CODE REVIEW

File: .github/workflows/cim-mapping-aws.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          image: ${{ steps.read_docker_compose.outputs.data }}
+          environment-variables: |
+            JWT_VERIFY_KEY=${{ secrets.JWT_VERIFY_KEY }}
+            JWT_VERIFY_ISSUER=${{ secrets.JWT_VERIFY_ISSUER }}
+
+      - name: Deploy to Amazon ECS service
+        uses: aws-actions/amazon-ecs-deploy-task-definition@v1
+        with:
+          task-definition: ${{ steps.render-cim-mapping.outputs.task-definition }}
+          service: Compas-Cim-Mapping
```
Comment (Stef3st) [inline .github/workflows/cim-mapping-aws.yml:48]: change it to `CoMPAS-Cim-Mapping` to keep consistency in our naming

**Review by Stef3st — CHANGES_REQUESTED** [review]
LGTM assuming the secrets are correct! I only miss a workflow for the validator, is this left out on purpose?

File: .github/workflows/cim-mapping-aws.yml (hunk 1 above)
Comment (pascalwilbrink) [reply .github/workflows/cim-mapping-aws.yml:48]: this is the name of how it's defined in AWS

**Comment by pascalwilbrink** [comment]
> LGTM assuming the secrets are correct! I only miss a workflow for the validator, is this left out on purpose?


Yes, the Validator won't be published on AWS

**Review by juancho0202 — APPROVED** [review]

**Review by juancho0202 — APPROVED** [review]

**Pull request merged**